src/xenia/ui/imgui_drawer.h

```diff
@@ -93,6 +93,8 @@ class ImGuiDrawer : public WindowInputListener, public UIDrawer {
   void LoadInputSystem(hid::InputSystem* input_system);
   void SetGuideButtonAction(std::function<void(uint8_t)> func);
 
+  size_t DialogsCount() const { return dialogs_.size(); };
```

Comment:
Usually there is no need to know exact amount of dialogs being opened. I suggest making "IsAnyDialogOpen"